go/datas/remote_database_handlers.go

```diff
@@ -252,6 +256,13 @@ func handleRootPost(w http.ResponseWriter, req *http.Request, ps URLParams, cs c
 	}
 }
 
+func handleBaseGet(w http.ResponseWriter, req *http.Request, ps URLParams, rt chunks.ChunkStore) {
+	d.PanicIfTrue(req.Method != "GET", "Expected get method.")
+
+	fmt.Fprintf(w, "<html><p>Hi. This is a Noms http server.</p><p>To learn more, visit <a href=\"https://github.com/attic-labs/noms\">our github project</a>.</p>")
```

Comment:
Capitalize HTTP. Also are there tests for the other endpoints? If so it would be good to add one for this so that it continues to work. You could make the string a constant and test that calling the root emits the constant.